jumanji/environments/packing/bin_pack/env.py

```diff
@@ -211,19 +219,36 @@ def observation_spec(self) -> specs.Spec[Observation]:
             }
         ems = specs.Spec(EMS, "EMSSpec", **ems_dict)
         ems_mask = specs.BoundedArray((obs_num_ems,), bool, False, True, "ems_mask")
+        items_axes = (
+            ["x_len", "y_len", "z_len", "value"]
+            if valued_items_used
+            else ["x_len", "y_len", "z_len"]
+        )
         if self.normalize_dimensions:
             items_dict = {
                 f"{axis}": specs.BoundedArray((max_num_items,), float, 0.0, 1.0, axis)
```

Comment:
You might get the same bug as the one you're speaking of [here](https://github.com/rsjeffers/jumanji_value_based/pull/4/files/5112be6fa8d3a16bd220236e49e6a93b6bd399be#r1267067438), so the value spec needs to be between -1 and 1 instead of 0 and 1.

Reply:
This is solved in the latest refactoring.